config/initializers/blacklight_initializer.rb

```diff
@@ -24,39 +24,57 @@ def export_as_openurl_ctx_kev(format = nil)
     id = to_marc.find{|field| field.tag == '001'}
```

Comment:
I don't know if this is ideal, but we've been opening the orangeindex issues in orangelight.